apiserver/failoverservice/failoverimpl.go

```diff
@@ -114,6 +115,11 @@ func QueryFailover(name, ns string) msgs.QueryFailoverResponse {
 
 	log.Debugf("query failover called for %s", name)
 
+	// indicate in the response whether or not a standby cluster
+	if cluster.Spec.Standby {
+		response.Standby = true
+	}
```

Comment:
`response.Standby = cluster.Spec.Standby`

Reply:
Changed to `cluster.Spec.Standby`.